http4k-format/moshi/src/main/kotlin/org/http4k/format/ConfigurableMoshi.kt

```diff
@@ -95,8 +104,11 @@ fun Moshi.Builder.asConfigurable() = object : AutoMappingConfiguration<Moshi.Bui
         }
 
     // add the Kotlin adapter last, as it will hjiack our custom mappings otherwise
-    override fun done() =
-        this@asConfigurable.add(KotlinJsonAdapterFactory()).add(Unit::class.java, UnitAdapter)
+    override fun done(): Moshi.Builder {
+        adapter(BiDiMapping(Double::toBigDecimal, BigDecimal::toDouble), { value(it) }, { nextDouble() })
```

Comment:
Since `MoshiNode` holds `BigDecimal` and `BigInteger`, I have to add a low-level adapter for them.  It feels weird, but I feel like it's the only way to be immune to floating-point imprecision.